department-of-veterans-affairs/va.gov-cms
# VACMS 2700 single alert added below page intro blocks

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2700

## Testing done


## Screenshots


## QA steps

As an admin

1. Login

2. Confirm the field placement and content creation experience matches the Acceptance Criteria below:
  - [x] new field "Alert" field_alert| entity revisions reference to "Alert (Single)" paragpraph only | cardinality single is added 
          to the following content types:
      - [x] FAQ - multiple Q&As (alert field located directly under "Title and introduction" fieldset)
      - [x] Checklist (alert field located directly under "Title and introduction" fieldset)
      - [x] Q&A (alert field located directly under "Question and Answer" fieldset)
      - [x] Step-by-step (alert field located directly under "Title and introduction" fieldset)
 - [x] node display configuration is adjusted to reflect the order of information on the form for each of the above content 
        types
 - [x] spec tool / content model tests are updated

3. Confirm Alert Single behaves as intended on each of the following content types node creation and edit:
- [x] FAQ
- [x] Checklist
- [x] Q&A
- [x] Step-by-step


## Definition of Done
